### PR TITLE
Add Community component

### PR DESCRIPTION
**Pull request by kytiken**
### Overview:概要
Add community component.

### CODE REVIEW

File: front/__tests__/components/Community.spec.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    const communityParams = {
+      name: 'shinosaka.rb',
+      description: 'This community is active in shinosaka.'
+    }
+
+    const component = renderer.create(
+      <Community community={ model(communityParams) } />
+    )
+
+    let tree = component.toJSON()
```
Comment (chimame) [inline front/__tests__/components/Community.spec.js]: Let's use `const`.

File: front/client/components/Community/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+
+export default class Community extends Component {
+  render() {
+    return (
+      <div>
+        <label>
```
Comment (chimame) [inline front/client/components/Community/index.js]: Do you use `label`? I think that `p` is good.

File: front/__tests__/components/Community.spec.js (hunk 1 above)
Comment (kytiken) [inline front/__tests__/components/Community.spec.js]: Sorry. I will be careful more careful.

File: front/client/components/Community/index.js (hunk 2 above)
Comment (kytiken) [inline front/client/components/Community/index.js]: I see. I will fix to delete the label.

```
<span>name: </span><p>Shinosaka.rb</p>
```
Like this?

Comment (kytiken) [inline front/client/components/Community/index.js]: Which do you think is better `p` tag or `span` tag?

Comment (chimame) [inline front/client/components/Community/index.js]: I think that you do not have to think hard.
```html
<p>name: Shinosaka.rb</p>
```
It is as easy as this.

**Comment by chimame** [comment]
>I think that you do not have to think hard.
>```<p>name: Shinosaka.rb</p>```
>It is as easy as this.

Was my feedback late?

File: front/client/components/Community/index.js (hunk 2 above)
Comment (kytiken) [inline front/client/components/Community/index.js]: I see.

**Comment by kytiken** [comment]
> Was my feedback late?

Not really. Sorry for my late reply.

**Comment by kytiken** [comment]
Thank you!!

**Pull request merged**